File: src/main/java/interfaces/TakaoLog.java
package interfaces;

import org.slf4j.Logger;
import org.slf4j.LoggerFactory;
import starter.Start;

public class TakaoLog {
	private static Logger logger = LoggerFactory.getLogger(Start.class);

	public static void logInfo(String message) {
		logger.info(message);
	}

	public static void logDebug(String message) {
		logger.debug(message);
	}

	public static void logError(String message) {
		logger.error(message);
	}
}

File: src/main/kotlin/it/discordbot/core/TakaoLog.kt
package it.discordbot.core

import org.slf4j.LoggerFactory
import org.springframework.stereotype.Component

/**
 * Logger Unico per il bot
 */
@Component
class TakaoLog {
	companion object {
		private var logger = LoggerFactory.getLogger(TakaoLog::class.java)

		fun logDebug(message: String) {
			logger.debug(message)
		}

		fun logInfo(message: String) {
			logger.info(message)
		}

		fun logError(message: String) {
			logger.error(message)
		}
	}
}